website/content/docs/agent/template.mdx

```diff
@@ -194,37 +205,17 @@ using the certificates `validTo` field.
 This does not apply to certificates generated with `generate_lease: true`. If set
 Vault Agent template will apply the non-renewable, leased secret rules.
 
-## Auto Auth and Templating Example
+When Agent's auto-auth triggers a re-authentication, e.g. due to token expiry,
+it generates a new token for Agent's use. This triggers a template server
+restart, which fetched and re-renders a new set of certificates even if existing
+certificates are valid.
+
+## Templating Example
 
 The following demonstrates configuring Vault Agent to template secrets using the
 AppRole Auth method:
```

Comment:
Would it be fine to update other examples to be like so as well? I stopped short of updating all the ones that I've found to be too long since I wasn't sure whether we'd want to keep a full config all the time.

Reply:
Go for it.  

If needed, we can add a note to say, "Refer to the [example configuration](/docs/agent#example-configuration) section for a complete Vault Agent configuration" or something like that. But short example looks cleaner.

Reply:
I went ahead and shortened a few more example snippets. On these examples I've added a comment section at the top to indicate that there may be other config blocks like so:
```python
# Other Vault Agent configuration blocks
# ...
```

Let me know what you think.

Reply:
Nice!